frontend/src/components/CountriesFilterSelect.tsx

```diff
@@ -14,7 +14,7 @@ export default function CountriesFilterSelect({
   onChange,
 }: CountriesFilterSelectProps) {
   const options = Object.fromEntries(
-    countries.map((country) => [country.code, country.label]),
+    countries.map((country) => [country.code, country.label]).sort(),
```

Comment:
```suggestion
    countries
      .map((country) => [country.code, country.label])
      .sort((a, b) => a[1].localeCompare(b[1])),
```

This doesn’t work reliably as it’s sorting by country code (which is different from the label that’s visible to users). For example, "Germany" would be listed before "Denmark". Instead, simply sort explicitly by label:

Reply:
Uff, yes. My bad. :disappointed:  Will fix and merge!